.github/workflows/ci.yaml

```diff
@@ -66,14 +70,31 @@ jobs:
           sudo rm -rf /usr/local
           df -h
       - uses: actions/checkout@v2
+      - name: Get rosdistro version for a given
+        id: ros_distro_version
+        shell: bash
+        env:
+          ROS_DISTRO: ${{ matrix.env.ROS_DISTRO }}
+        run: |
+          latest-release-date() {
+              echo "$(curl -sL "https://api.github.com/repos/ros/rosdistro/tags?per_page=100" \
+              | grep -o "$1/[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}" \
+              | cut -d'/' -f2 \
+              | uniq \
+              | sort -t- -k1n -k2n -k3n -z | head -n 1)"
+          }
+          ROS_DISTRO_VERSION=`latest-release-date $ROS_DISTRO`
+          echo "The latest version for $ROS_DISTRO is $ROS_DISTRO_VERSION"
+          echo set-output name=ROS_DISTRO_VERSION::${ROS_DISTRO_VERSION}
+          echo ::set-output name=ROS_DISTRO_VERSION::${ROS_DISTRO_VERSION}
```

Comment:
What does the :: prefix do and why do we need both of these lines?

Reply:
The first one is just a debugging message, the second one for setting a step output value